starknet_py/tests/e2e/docs/quickstart/test_using_account_client.py

```diff
@@ -0,0 +1,47 @@
+import os
+from pathlib import Path
+import pytest
+from starknet_py.tests.e2e.utils import DevnetClientFactory
+
+directory = os.path.dirname(__file__)
+map_source_code = Path(directory, "map.cairo").read_text("utf-8")
+
+
+@pytest.mark.asyncio
+async def test_using_account_client(run_devnet):
```

Comment:
For an unknown reason, this test takes much more time than others. any ideas?

Reply:
You're deploying an Account to testnet in line 19 which is much slower that devnet. Perhaps there's a way to somehow include this line in docs but skip actual deployment?